index.html

```diff
@@ -13,11 +13,36 @@
     -->
 
 	<!-- Meta -->
-	<title>Dogecoin</title>
+	<title>Dogecoin - #dogecoin - #doge</title>
 	<meta charset="utf-8">
-	<meta content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide." name="description">
 	<meta content="width=device-width, initial-scale=1.0" name="viewport">
 
+	<!-- og:properties -->
+	<meta property="og:site_name" name="dogecoin.com" />
+	<meta property="og:type" content="website" />
+	<meta property="og:description" content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide.">
+	<meta property="og:title" content="dogecoin built on the principles of sound money - a transactional currency made for the internet" >
+	<meta property="og:url" content="https://dogecoin.com" />
+
+	<!-- og:image -->
+	<meta property="og:image" content="/imgs/doge-ogimage.png" />
+	<meta property="og:image:type" content="image/png" />
+	<meta property="og:image:width" content="1024" />
+	<meta property="og:image:height" content="1270" />
+	<meta property="og:image:alt" contect="dogecoin.com Shiba Inu" />
+
+	<!-- Twitter Card -->
+	<meta name="twitter:card" content='summary_large_image' />
+	<meta name="twitter:site" content="@dogecoin" />
+	<meta name="twitter:creator" content="@dogecoin_dev" />
+	<meta name="twitter:title" content="#dogecoin a transactional currency built for the internet" />
+	<meta name="twitter:description" content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide." />
+	<meta name="twitter:domain" content='https://dogecoin.com' />
+	<meta name="twitter:image" content="/imgs/doge-ogimage.png" />
+	<meta name="twitter:image:alt" contect="dogecoin.com Shiba Inu" />
+	<!--- favicon -->
+	<link rel="shortcut icon" href="../imgs/avatar.png" />
```

Comment:
Any reason for this using a relative path contrary to all the others?

Reply:
its wrong my mistake..
fixed relative path